client/components/embed-container/index.jsx

```diff
@@ -57,6 +59,69 @@ function embedInstagram( domNode ) {
 	);
 }
 
+let twitterLoader;
+function embedTwitter( domNode ) {
+	debug( 'processing twitter for', domNode );
+	if ( domNode.hasAttribute( 'data-wpcom-embed-processed' ) ) {
+		return; // already marked for processing
+	}
+
+	domNode.setAttribute( 'data-wpcom-embed-processed', '1' );
+
+	if ( typeof twttr !== 'undefined' ) {
+		global.twttr.widgets.load( domNode );
+		return;
+	}
+
+	if ( ! twitterLoader ) {
+		twitterLoader = new Promise( function( resolve, reject ) {
+			loadScript( 'https://platform.twitter.com/widgets.js', function( err ) {
+				if ( err ) {
+					reject( err );
+				} else {
+					resolve();
+				}
+			} );
+		} );
+	}
+
+	twitterLoader.then(
+		embedTwitter.bind( null, domNode ),
+		debug.bind( null, 'Could not load twitter platform' )
+	);
+}
+
+let fbLoader;
+function embedFacebook( domNode ) {
+	debug( 'processing twitter for', domNode );
+	if ( domNode.hasAttribute( 'data-wpcom-embed-processed' ) ) {
+		return; // already marked for processing
+	}
+
+	domNode.setAttribute( 'data-wpcom-embed-processed', '1' );
```

Comment:
they do, but it's "slower". see https://bugzilla.mozilla.org/show_bug.cgi?id=802157